src/pages/deploy/index.tsx

```diff
@@ -26,7 +26,13 @@ const PastDeployLogsPage = () => {
           deploy, end deploy and state.
         </p>
       </div>
-      <DeployLogs deployTab={pastDeploymentsRichList} />
+      {pastDeploymentsRichList.length != 0 ? (
```

Comment:
You inverted the condition:

```suggestion
      {pastDeploymentsRichList.length === 0 ? (
```